tests/Microsoft.Identity.Test.Common/AuthorityWithExpectedTenantId.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Identity.Test.Common
+{
+    public struct AuthorityWithExpectedTenantId
```

Comment:
Why struct and not class?

Reply:
I did not even notice. I think the documentation that I used had a struct. Will change to class, should not matter

Reply:
Done